pkg/controller/replication/replication_controller_test.go

```diff
@@ -894,37 +897,11 @@ func TestDeleteControllerAndExpectations(t *testing.T) {
 
 	// This should have no effect, since we've deleted the rc.
 	podExp.Add(-1, 0)
-	manager.podStore.Indexer.Replace(make([]interface{}, 0), "0")
+	manager.podLister.Indexer.Replace(make([]interface{}, 0), "0")
 	manager.syncReplicationController(getKey(rc, t))
 	validateSyncReplication(t, &fakePodControl, 0, 0, 0)
 }
 
-func TestRCManagerNotReady(t *testing.T) {
```

Comment:
This test doesn't make sense anymore here since the rc manager is not responsible now for starting any of the informers. Funny enough, it's the only respective test that is missing from the replica set unit tests.